[2021-05-14 21:10:06] local.ERROR: There is no existing directory at "/opt/lampp/htdocs/dil/cozyAppartment/storage/logs" and it could not be created: Permission denied {"exception":"[object] (UnexpectedValueException(code: 0): There is no existing directory at \"/opt/lampp/htdocs/dil/cozyAppartment/storage/logs\" and it could not be created: Permission denied at /opt/lampp/htdocs/dil/df/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php:172)
[stacktrace]
#0 /opt/lampp/htdocs/dil/df/vendor/monolog/monolog/src/Monolog/Handler/StreamHandler.php(100): Monolog\\Handler\\StreamHandler->createDir()
#1 /opt/lampp/htdocs/dil/df/vendor/monolog/monolog/src/Monolog/Handler/AbstractProcessingHandler.php(42): Monolog\\Handler\\StreamHandler->write(Array)
#2 /opt/lampp/htdocs/dil/df/vendor/monolog/monolog/src/Monolog/Logger.php(317): Monolog\\Handler\\AbstractProcessingHandler->handle(Array)
#3 /opt/lampp/htdocs/dil/df/vendor/monolog/monolog/src/Monolog/Logger.php(539): Monolog\\Logger->addRecord(400, 'There is no exi...', Array)
#4 /opt/lampp/htdocs/dil/df/vendor/laravel/framework/src/Illuminate/Log/Logger.php(174): Monolog\\Logger->error('There is no exi...', Array)
#5 /opt/lampp/htdocs/dil/df/vendor/laravel/framework/src/Illuminate/Log/Logger.php(87): Illuminate\\Log\\Logger->writeLog('error', 'There is no exi...', Array)
#6 /opt/lampp/htdocs/dil/df/vendor/laravel/framework/src/Illuminate/Log/LogManager.php(547): Illuminate\\Log\\Logger->error('There is no exi...', Array)
#7 /opt/lampp/htdocs/dil/df/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(123): Illuminate\\Log\\LogManager->error('There is no exi...', Array)
#8 /opt/lampp/htdocs/dil/df/app/Exceptions/Handler.php(39): Illuminate\\Foundation\\Exceptions\\Handler->report(Object(UnexpectedValueException))
#9 /opt/lampp/htdocs/dil/df/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(403): App\\Exceptions\\Handler->report(Object(UnexpectedValueException))
#10 /opt/lampp/htdocs/dil/df/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(111): Illuminate\\Foundation\\Http\\Kernel->reportException(Object(UnexpectedValueException))
#11 /opt/lampp/htdocs/dil/df/public/index.php(55): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#12 /opt/lampp/htdocs/dil/df/server.php(21): require_once('/opt/lampp/htdo...')
#13 {main}
"} 
